src/data/map.js

```diff
@@ -1270,14 +1291,23 @@ const getAvailableRaidBosses = async () => {
 };
 
 const getAvailableQuests = async () => {
-    //const rewards = ['stardust']; // TODO: Localize
     let sql = 'SELECT quest_item_id AS id FROM pokestop WHERE quest_reward_type=2 GROUP BY quest_item_id';
     const itemResults = await db.query(sql);
     sql = 'SELECT quest_pokemon_id AS id FROM pokestop WHERE quest_reward_type=7 GROUP BY quest_pokemon_id';
     const pokemonResults = await db.query(sql);
+    sql = `
+    SELECT
+        quest_pokemon_id AS id,
+        json_extract(json_extract(quest_rewards, '$[*].info.amount'), '$[0]') AS amount
+    FROM rdmdb.pokestop
+    WHERE quest_reward_type = 12
+    GROUP BY quest_pokemon_id
```

Comment:
```suggestion
    SELECT
        DISTINCT json_extract(json_extract(quest_rewards, '$[*].info.pokemon_id'), '$[0]') AS id
    FROM pokestop
    WHERE quest_reward_type = 12
    GROUP BY id;
```

Reply:
With `DISTINCT` I think we can drop the `GROUP BY id` clause.